Aptoide/appcoins-wallet-android
# [feature/APPC-1251] - Update settings screen

**What does this PR do?**

This PR implements the updated settings screen.
New layout: https://zpl.io/adN6OG5

**Database changed?**

   No

**Where should the reviewer start?**

- SettingsActivity.java
- SettingsFragment.kt

**How should this be manually tested?**

  By opening every setting and see if the result is the expected

  Flow on how to test this or QA Tickets related to this use-case: [APPC-1251](https://aptoide.atlassian.net/browse/APPC-1251)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-1251](https://aptoide.atlassian.net/browse/APPC-1251)

**Questions:**

   The "Verify Wallet" preference is still not finished, the flow on the wallet verification has not been reviewed yet. This PR should only be merged after this preference is completed.



**Code Review Checklist**
